features/step_definitions/api_request_steps.rb

```diff
@@ -243,6 +243,16 @@
   }]
 end
 
+Given("I add {string} of multiple regular_transactions, of {float} per month of debit") do |category, amount|
```

Comment:
can these 2 steps be turned into a single step with the same title and we pass in `debit` or `credit`
```
Given("I add {string} of multiple regular_transactions, of {float} per month of debit") do |category, amount, type|  @regular_transactions ||= []
  @regular_transactions << {
    category:,
    frequency: "monthly",
    operation: type,
    amount:,
  }